wakuv2/common/message.go

```diff
@@ -166,7 +166,7 @@ func NewReceivedMessage(env *protocol.Envelope, msgType MessageType) *ReceivedMe
 	return &ReceivedMessage{
 		Envelope:     env,
 		MsgType:      msgType,
-		Sent:         uint32(env.Message().Timestamp / int64(time.Second)),
+		Sent:         uint32(*env.Message().Timestamp / int64(time.Second)),
```

Comment:
Altho it will never happen at this point, the timestamp is an optional field in the protobuffer. Using `GetTimestamp()` will ensure you get at least a `0` in case for some reason in the future the usage for timestamp changes.

```suggestion
		Sent:         uint32(env.Message().GetTimestamp() / int64(time.Second)),
```